docs/hour-of-code-2022.html

```diff
@@ -0,0 +1,396 @@
+        />
+        <!-- <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1.0, user-scalable=yes"
+        /> -->
+
+        <link rel="manifest" href="/blb/sim.webmanifest" />
+
+        <meta name="mobile-web-app-capable" content="yes" />
+        <meta name="apple-mobile-web-app-capable" content="yes" />
+        <meta
+            name="apple-mobile-web-app-title"
+            content="Microsoft MakeCode Arcade Hour of Code™ 2022"
+        />
+        <meta
+            name="apple-mobile-web-app-status-bar-style"
+            content="translucent-black"
+        />
+        <meta name="format-detection" content="telephone=no" />
+        <meta
+            name="application-name"
+            content="Microsoft MakeCode Arcade Hour of Code™ 2021"
+        />
+
+        <meta name="msapplication-TileColor" content="#3DC1AA" />
+        <meta
+            name="msapplication-TileImage"
+            content="static/hour-of-code/2022/hoc-card.png"
+        />
+        <meta name="msapplication-config" content="none" />
+        <meta name="theme-color" content="#3DC1AA" />
+
+        <meta name="twitter:card" content="summary" />
+        <meta name="twitter:site" content="MSMakeCode" />
+        <meta
+            name="twitter:title"
+            content="Microsoft MakeCode Arcade Hour of Code™ 2021"
+        />
+        <meta
+            name="twitter:description"
+            content="Join us in celebrating this year's Computer Science Education Week by playing, designing and coding your very own retro arcade games!"
+        />
+
+        <meta
+            name="twitter:image"
+            content="static/hour-of-code/2021/hoc-card.png"
+        />
+
+        <meta
+            property="og:title"
+            content="Microsoft MakeCode Arcade Hour of Code™ 2021"
+        />
+        <meta
+            property="og:site_name"
+            content="Microsoft MakeCode Arcade Hour of Code™ 2021"
+        />
+        <meta
+            property="og:description"
+            content="Join us in celebrating this year's Computer Science Education Week by playing, designing and coding your very own retro arcade games!"
+        />
+
+        <meta
+            property="og:image"
+            content="static/hour-of-code/2021/hoc-card.png"
+        />
+
+        <link
+            rel="apple-touch-icon"
+            href="static/hour-of-code/2021/favicon.png"
+        />
+        <link
+            rel="icon"
+            type="image/png"
+            href="static/hour-of-code/2021/favicon.png"
+        />
+        <link
+            rel="shortcut icon"
+            href="static/hour-of-code/2021/favicon.png"
+        />
+
+        <!-- <link
+            rel="stylesheet"
+            href="https://pxt.azureedge.net/blob/2163189fd5e35c0981ed55318415582a7c9aeb12/doccdn/semantic.css"
+            type="text/css"
+        /> -->
+        
+        <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
we can't do external fonts like this, need to pack them in; e.g. this is how it was done in 2021's hoc https://github.com/microsoft/pxt-arcade/pull/4181/commits/2650bebcb69b13df32c90ac61ed3c0ac13e4252e

Reply:
The comment said "legally approved", do I need to get this font approved by someone?